.github/workflows/pypi.yml

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: RayDP PyPi
+
+on: 
+  schedule:
+    - cron: '0 0 * * *'
+  # can manually trigger the workflow
+  workflow_dispatch:
+
+jobs:
+  build-and-publish:
+    # do not run in forks
+    if: ${{ startsWith(github.ref, 'refs/tags') && (github.repository_owner == 'oap-project') }}
+    name: build wheel and upload
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Build wheel
+      env:
+        RAYDP_PACKAGE_NAME: raydp_nightly
+      run: pip install wheel && ./build.sh
```

Comment:
Does this work without java?

Reply:
Oh, I forgot it. But it works in my repo, weird. Maybe it did not use java 1.8. I'll add the java part anyway.